site/docs/master/style-guide.md

```diff
@@ -0,0 +1,317 @@
+
+<table caption="Do and Don't - Starting a sentence with a component tool or component name" >
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td markdown="span">The `kubeadm` tool bootstraps and provisions machines in a cluster.</td><td markdown="span">`kubeadm` tool bootstraps and provisions machines in a cluster.</td></tr>
+<tr><td>The kube-scheduler is the default scheduler for Kubernetes.</td><td>kube-scheduler is the default scheduler for Kubernetes.</td></tr>
+</table>
+
+### Use normal style for string and integer field values
+
+For field values of type string or integer, use normal style without quotation marks.
+
+<table caption="Do and Don't - Use normal style for string and integer field values" >
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td markdown="span">Set the value of `imagePullPolicy` to `Always`.</td><td markdown="span">Set the value of `imagePullPolicy` to "Always".</td></tr>
+<tr><td markdown="span">Set the value of `image` to `nginx:1.16`.</td><td markdown="span">Set the value of `image` to nginx:1.16.</td></tr>
+<tr><td markdown="span">Set the value of the `replicas` field to `2`.</td><td markdown="span">Set the value of the `replicas` field to 2.</td></tr>
+</table>
+
+## Code snippet formatting
+
+
+### Don't include the command prompt
+
+<table caption="Do and Don't - Don't include the command prompt" >
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td>kubectl get pods</td><td>$ kubectl get pods</td></tr>
+</table>
+
+### Separate commands from output
+
+Verify that the Pod is running on your chosen node:
+
+```
+kubectl get pods --output=wide
+```
+
+The output is similar to this:
+
+```
+NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
+nginx    1/1       Running   0          13s    10.200.0.4   worker0
+```
+
+## Velero.io word list
+
+
+A list of Velero-specific terms and words to be used consistently across the site.
+
+<table caption="Velero.io word list" >
+<tr><th>Trem</th><th>Useage</th></tr>
+<tr><td>Kubernetes</td><td>Kubernetes should always be capitalized.</td></tr>
+<tr><td>Docker</td><td>Docker should always be capitalized.</td></tr>
+<tr><td>Velero</td><td>Velero should always be capitalized.</td></tr>
+<tr><td>VMware</td><td>VMware should always be correctly capitalized.</td></tr>
+<tr><td>On-premises</td><td>On-premises or on-prem rather than on-premise or other variations.</td></tr>
+<tr><td>Backup</td><td>Backup rather than back up, back-up or other variations.</td></tr>
+<tr><td>Plugin</td><td>Plugin rather than plug-in or other variations.</td></tr>
+</table>
+
+## Markdown elements
+
+
+### Line breaks
+Use a single newline to separate block-level content like headings, lists, images, code blocks, and others. The exception is second-level headings, where it should be two newlines. Second-level headings follow the first-level (or the title) without any preceding paragraphs or texts. A two line spacing helps visualize the overall structure of content in a code editor better.
```

Comment:
This is a personal preference, but I line break at sentences in Markdown to make content easier to review. Curious what others think about that practice; clearly this isn't how it's viewed in the Kubernetes docs team.

Reply:
I think this direction is overkill. Mostly we follow what's already there, and this is only for consumption of whoever is reading the markdown code, not for users of the documentation. I'd remove this one.

Reply:
To nolan's point, the goal of line break rules is to make markdown easier to read and update. Do you mostly review docs in Github or using the netlify preview? To me this rule seems like too much, but if the team felt strongly about having character limits and line breaks done a certain way we can change it to something else

Reply:
I do both - I like GitHub's diff view for seeing what changed, and netlify's preview for how it renders.

I am fine with dropping the line break guideline entirely.

Reply:
Ok removed the line break rule